src/expression.c

```diff
@@ -9571,7 +9573,10 @@ Expression *DeleteExp::semantic(Scope *sc)
                 FuncDeclaration *fd = sd->dtor;
 
                 if (!f)
+                {
+                    semanticTypeInfo(sc, ts);
```

Comment:
These implicit dependencies are really the worst maintenance wise?
Is there any way we can do this in the glue code where the TypeInfo is actually requested?